### PR TITLE
add common ctf import protocol

### PR DESCRIPTION
**Pull request by azazellochg**
Similar to SPA, the idea is to have one import CTF protocol that will call the code from different plugins.
Related PRs: 
https://github.com/scipion-em/scipion-em-cistem/pull/74
https://github.com/scipion-em/scipion-em-imod/pull/243

### CODE REVIEW

File: tomo/protocols/protocol_import_ctf.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+                    logger.info("Parsing file: " + defocusFn)
+
+                    newCTFTomoSeries = CTFTomoSeries()
+                    newCTFTomoSeries.copyInfo(ts)
+                    newCTFTomoSeries.setTiltSeries(ts)
+                    newCTFTomoSeries.setObjId(tsObjId)
+                    newCTFTomoSeries.setTsId(tsId)
+
+                    if self._importFromImod():
+                        # Create IMOD-specific attrs that will be updated later
```
Comment (pconesa) [inline tomo/protocols/protocol_import_ctf.py:111]: Can this be done inside the Import class from imod?

Comment (azazellochg) [reply tomo/protocols/protocol_import_ctf.py:111]: that was my initial idea, but then I'd have to put outputCtfs.append(newCTFTomoSeries) inside ci.parseTSDefocusFile() for each plugin, which is not very clean. I could not append the item to a set and then add more attrs to the same item

Comment (JorMaister) [reply tomo/protocols/protocol_import_ctf.py:111]: A notification to take into consideration: until this is merged, as the protocol for importing tomo CTFs in Cistem plugin is out after the last PR, we are not offering (in devel) a protocol to import the tomo CTFs from Cistem and the corresponding tests won't work.

Comment (azazellochg) [reply tomo/protocols/protocol_import_ctf.py:111]: @pconesa I'm open to suggestions if you have any workarounds..

**Comment by azazellochg** [comment]
Any update here? I cannot merge imod PR without this one first.

**Pull request merged**